### DISCUSSION TITLE
Can this plugin become Obsidian Publish Alternative?

### DISCUSSION THREAD

**Discussion by stojan97** [alangrainger/share-note#48] [Vote for upcoming features]
Since I dabbled through all free alternatives for Obsidian Publish, ([Digital Garden](https://github.com/oleeskild/obsidian-digital-garden), [Obsidian WebExport](https://github.com/KosmosisDire/obsidian-webpage-export)), they all seem to fail at the thing that this plugin is not.

**That is, converting the Markdown files to HTML exactly as they are as in Obsidian.** 

Can we make this plugin as simple as possible, with **title at the top left corner**, **navigation bar** on the left, and **outline** (table of contents) on the right, with simple customization of the width and color for them or have them as the theme by default.

I'm planning to do something like this that, but when I saw this plugin I was surprised at how on point are the notes with the ones in Obsidian no matter the plugin.

I would also suggest to support **Folder notes**, to have index pages since that is very important lately in documentation pages.

 
![simple_webpage](https://github.com/alangrainger/share-note/assets/10180981/da1821b1-b1bb-4b00-8ca7-dbf674907d6c)

**Comment by alangrainger** [comment]
Hi, I appreciate the suggestion, but it's not in the scope of this plugin to be a Publish alternative. This plugin is specifically for sharing individual notes, not entire vaults.

As you say:

> Can we make this plugin as simple as possible

That's the current goal, and going for the full Publish experience would be heading away from that direction.
